### PR TITLE
Ability to pick options values from env vars on gitlab_runners inventory plugin

### PR DESCRIPTION
**Pull request by kalw**
Add env vars capability for server_url and api_token

##### SUMMARY

Permit env var input for server_url and api_token options values 

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

plugins/inventory/gitlab_runners
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/plugins/inventory/gitlab_runners.py [hunk 1]
```diff
@@ -31,11 +31,15 @@
               - gitlab_runners
         server_url:
             description: The URL of the GitLab server, with protocol (i.e. http or https).
+            env:
+              - name: GITLAB_SERVER_URL
```
Comment (felixfontein) [inline lib/ansible/plugins/inventory/gitlab_runners.py:35]:
```suggestion
              - name: GITLAB_SERVER_URL
                version_added: '2.10'
```
Same for the other option.

**Review by felixfontein — COMMENTED** [review]
You also need a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to) here.

**Comment by felixfontein** [comment]
CC @dj-wasabi @Lunik @marwatk @Shaps @waheedi @jlozadad

**Comment by Akasurde** [comment]
@kalw Are you still working on this? If yes, Could you please migrate this PR to the[`community.general`](https://github.com/ansible-collections/community.general) collection repo? 

If you have further questions please stop by IRC or the mailing list:

* IRC: #ansible on irc.freenode.net
* mailing list: https://groups.google.com/forum/#!forum/ansible-project

Thanks,

needs_info

**Comment by kalw** [comment]
modifications has been moved and merged into https://github.com/ansible-collections/community.general/pull/611#issuecomment-659221545

**Pull request closed without merging**